attributes/default.rb

```diff
@@ -17,13 +17,21 @@
 # limitations under the License.
 #
 
-default['mcafee']['agent']['package_name']          = 'McAfee Agent'
-default['mcafee']['agent']['url']                   = nil
-default['mcafee']['agent']['checksum']              = nil
-default['mcafee']['agent']['options']               = '/Install=Updater /Silent'
+default['mcafee']['agent']['package_name']            = 'McAfee Agent'
+default['mcafee']['agent']['url']                     = nil
+default['mcafee']['agent']['checksum']                = nil
+default['mcafee']['agent']['options']                 = '/Install=Updater /Silent'
 
-default['mcafee']['virusscan']['package_name']      = 'McAfee VirusScan Enterprise'
-default['mcafee']['virusscan']['url']               = nil
-default['mcafee']['virusscan']['checksum']          = nil
-default['mcafee']['virusscan']['options']           = 'ADDLOCAL=ALL RUNAUTOUPDATESILENTLY=True REMOVE=ScriptScan,LotusNotesScan,EmailScan REBOOT=R /qn'
-default['mcafee']['virusscan']['installer_timeout'] = 900
+default['mcafee']['virusscan']['package_name']        = 'McAfee VirusScan Enterprise'
+default['mcafee']['virusscan']['url']                 = nil
+default['mcafee']['virusscan']['checksum']            = nil
+default['mcafee']['virusscan']['options']             = 'ADDLOCAL=ALL RUNAUTOUPDATESILENTLY=True REMOVE=ScriptScan,LotusNotesScan,EmailScan REBOOT=R /qn'
+default['mcafee']['virusscan']['installer_timeout']   = 900
+default['mcafee']['virusscan']['installer_exe']       = 'SetupVSE.exe'
+
+# If you want to install the newer McAfee Endpoint Security overwrite these virusscan attributes like this:
+# overwrite['mcafee']['virusscan']['package_name']      = 'McAfee Endpoint Security Platform'
+# overwrite['mcafee']['virusscan']['url']               = <url to the zip file>
+# overwrite['mcafee']['virusscan']['checksum']          = <corresponding checksum>
+# overwrite['mcafee']['virusscan']['options']           = 'ADDLOCAL="tp" /qn /nocontentupdate'
+# overwrite['mcafee']['virusscan']['installer_exe']     = 'setupEP.exe'
```

Comment:
Same here... Don't override them, just set them to the other values...

Reply:
also fixed